app/wp-content/themes/kstg/library/scss/style.scss

```diff
@@ -4,6 +4,7 @@
 @import "components/base";
 @import "components/typography";
 @import "components/header";
+@import "components/home-page";
 @import "components/footer";
```

Comment:
Maybe this is something you do on another branch, or lump into this one, but I would organize these a little better to keep things organized when things get bigger. Not everything is a 'component', right? It's kind of up to you as far as naming convention, but I would put the variables, typography, base and reset into a grouping that might be called something like 'core.' And then maybe grid would be a component? I think of a component as something that's repeated, but not necessarily a requirement (like variables or typography). And then maybe another directory called 'layout' for header, footer, home-page, etc.

[Here's a good example](https://github.com/derrickshowers/learn-me/tree/master/app/styles) of how we set things up for learn-me.

Call it whatever makes most sense to you, just nice to separate things out a bit more.